sunflower-land/sunflower-land
# [FEAT] Is tradeable

# Description

Adds separate dates for when items should can be traded (in the marketplace) or withdrawn.

Includes some helpful labels so players know when an item is tradeable or withdrawable.

<img width="419" alt="Screenshot 2024-12-13 at 3 09 59 PM" src="https://github.com/user-attachments/assets/4bd0cf70-e25f-4c3f-a1ce-60c9c720d16b" />
<img width="423" alt="Screenshot 2024-12-13 at 3 09 08 PM" src="https://github.com/user-attachments/assets/f6455cb8-829d-45a9-9399-3d36c099d687" />


## How to test?

1. Connect to local API
2. Explore the marketplace and ensure only trade-able items are there
3. Go to withdraw collectibles + wearables and ensure the correct ones are showing

_Note: By default the items not ready for trading will not show in the marketplace list. You can manually change the URL using these item IDs to visit them though._
